style.css

```diff
@@ -124,8 +119,267 @@ img {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 2.4rem;
+  margin-bottom: 80px;
+}
+
+.work-title-header {
+  margin-top: 100px;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: var(--font-round);
+  font-style: normal;
+}
+
+.title-separator {
+  width: calc(100% - 0px / 2 + 0.5px);
+}
+
+.gym-fit-img {
+  width: 100%;
+}
+
+.work-card-post {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.post-header {
+  font-size: 32px;
+  font-family: var(--font-round);
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  color: var(--font-blue);
+}
+
+.work-post-description {
+  color: var(--font-blue);
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.work-tag-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+}
+
+.work-items {
+  box-sizing: border-box;
+  padding: 10px 12px;
+  background: var(--font-white);
+  border: 1px solid #8993a4;
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: var(--font-blue);
+}
+
+.project-button {
+  font-family: var(--font-inter);
+  color: var(--font-white);
+  background: var(--font-orange);
+  padding: 12px;
+  gap: 10px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  margin-bottom: 20px;
+}
+
+.work-card-holder {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 24px;
+}
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+  background-image: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(./images/Img-Placeholder-2.png);
+  background-size: cover;
+  padding-top: 72px;
+}
+
+.info-card-holder {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 8px 16px;
+  gap: 12px;
+  color: var(--font-white);
+}
+
+.work-project-header {
+  font-family: var(--font-round);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.work-card-description {
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.work-tag-items {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  background-color: rgba(255, 255, 255, 0.24);
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  outline: none;
+}
+
+.work-button-section {
+  margin-top: 22px;
+}
+
+.card-button {
+  padding: 13px 0;
+  background: var(--font-orange);
+  cursor: pointer;
+  font-family: var(--font-inter);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: 0.03em;
+}
```

Comment:
- [ ] Good work applying styles to your buttons ✔️. However, please kindly ensure to add the changes on hover and while pressed provided in the Figma design 🤓.

Please kindly check how the buttons should look below 👇🏽

![buttons](https://user-images.githubusercontent.com/116701485/235292114-e430d88d-ed20-4211-a8d3-e05f8f4d39cf.png)